package.json

```diff
@@ -31,6 +33,8 @@
     "sentry-release": "SENTRY_RELEASE=$(git rev-parse --short HEAD); sentry-cli releases --log-level=debug new -p marketplace-apps $SENTRY_RELEASE && sentry-cli releases --log-level=debug set-commits --auto --ignore-missing $SENTRY_RELEASE && sentry-cli releases --log-level=debug finalize $SENTRY_RELEASE && sentry-cli releases --log-level=debug deploys $SENTRY_RELEASE new -e production"
   },
   "dependencies": {
+    "contentful-management": "^11.52.0",
+    "dotenv": "^16.0.0",
```

Comment:
If these are dependencies intended for the scripts folder, adding them here at the top-level project package.json probably isn't the right place. These should probably be added to `scripts/package.json` instead